src/main/resources/data-repository-openapi.yaml

```diff
@@ -143,6 +163,49 @@ paths:
         403:
           description: No permission to create studies
 
+  '/api/custody/v1/study/{id}':
+    get:  ## retrieve
+        description: Retrieve a study by id
+        operationId: retrieveStudy
+        tags:
+        - model
+        parameters:
+        - in: body
+          name: study
+          description: Id of study to retrieve
+          schema:
+            $ref: '#/definitions/UniqueIdModel'
+        responses:
+          200:
+            description: Created study
+            schema:
+              $ref: '#/definitions/StudyModel'
+          400:
+            description: Bad request - duplicate name, invalid name, badly formed schema
```

Comment:
This error description is wrong, I think. Probably should be a 404 - not found - if the id doesn't match anything.

There is always the question about whether we should distinguish 404 and 403; it gives out info about the existence of a study (or dataset) that someone does not have access to. My opinion for this API is that we should distinguish them. If we use API keys, then most users will not have access to the API in the first place, so we don't have the problem of random people probing for data they should not see. Further, I think our focus should be on helping the steward and custodian get the permissions right, rather than being strict about information hiding.